### PR TITLE
MONGOID-5656 Rails specs ought to enable FLE

### PR DESCRIPTION
**Pull request by jamis**
The Rails specs in the Mongoid evergreen suite do not enable FLE for the database rake tasks, which means those specs are consistently being skipped.

### CODE REVIEW

File: lib/mongoid/tasks/database.rb [hunk 1]
```diff
@@ -26,6 +26,9 @@ def create_collections(models = ::Mongoid.models, force: false)
           else
             logger.info("MONGOID: collection options ignored on: #{model}, please define in the root model.")
           end
+        rescue Exception
+          puts "error while creating collection for #{model}"
```
Comment (comandeo) [inline lib/mongoid/tasks/database.rb:30]: We use `logger` above, should we use it here, too?

**Review by comandeo — APPROVED** [review]
Great job, thank you!

File: lib/mongoid/tasks/database.rb (hunk 1 above)
Comment (jamis) [reply lib/mongoid/tasks/database.rb:30]: For whatever reason, the logger output doesn't display in CI, which is where I had to look to troubleshoot this initially. I can dig in and see why the logger output isn't being displayed; if I can figure that out, then I'll switch the `puts` to the logger here.

**Comment by johnnyshields** [comment]
@jamis thank you for this 🎉

**Pull request merged**